src/extension.ts

```diff
@@ -206,19 +207,32 @@ class AmigaDebugExtension {
 		if (process.platform !== "win32") {
 			await this.setPermissions();
 		}
+
+		if (vscode.extensions.getExtension('gigabates.m68k-lsp')) {
```

Comment:
This code creates a `settings.json` file even if I just open a folder to profile a `.uss` file. That folder doesn't contain anything else besides `.uss` files. Can we restrict the creation of the `settings.json` file? maybe only if there's an `amiga.json` file or something like this?

Reply:
Yeah that is a bit annoying. I'll have a look into it.